db/migrate/20240412111707_allow_long_webhook_types.rb

```diff
@@ -0,0 +1,9 @@
+class AllowLongWebhookTypes < ActiveRecord::Migration[7.0]
+  def up
+    change_column :subscriber, :webhook_url, :string, limit: 250
```

Comment:
as a url can be up to 2048 chars (approx) should we just make this a text field.....